src/content/docs/infrastructure/host-integrations/host-integrations-list/dojo-io-integration.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Dojo.io integration
+tags:
+    - dojo.io integration
+    - dojo integration
+    - new relic integrations
+metaDescription: "The New Relic Dojo.io integration sends performance metrics and inventory data from your Dojo.io framework to the New Relic platform."
+---
+import dojodashboard from 'images/infrastructure_screenshot-crop_dojo-dashboard.png'
+
+With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, user time on the site, and browser performance metrics like Initial page load, route change, throughput and page views with Javascript errors. Built with our infrastructure agent and our browser monitoring, the Dojo dashboard lets you view all your most important data in one place.
+
+<img
+  title="dashboard-example.png"
+  alt="A screenshot of the Dojo dashboard"
+  src={dojodashboard}
+/>
+
+## Install the infrastructure agent and Dojo integration [#install]
+
+1. Install the infrastructure agent (optional)
+Before getting Dojo data into New Relic, first install our infrastructure agent to get details about your host.
+* Follow our [guided install](https://one.newrelic.com/nr1-core?state=999174fa-ed8b-d707-af7b-8e4f8fd03cc1) to instrument your system with the infrastructure agent. Or if you prefer, you can install the infrastructure agent by [following the docs](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent).
```

Comment:
Steps 1 and 2 could probably be combined into one. We should probably include a sentence about why the infrastructure agent is optional. Since the browser agent hooks into Dojo to ingest data, infrastructure agent provides an added layer so you can monitor your entire system... but it's not necessary for the browser agent to work with Dojo.